src/pages/signin/SignInPage.js

```diff
@@ -61,8 +62,13 @@ const defaultProps = {
 class SignInPage extends Component {
     componentDidMount() {
         Performance.measureTTI();
-
         App.setLocale(Localize.getDevicePreferredLocale());
+        BackButtonUtils.prepareBackHistory();
+        BackButtonUtils.addBackButtonListener(BackButtonUtils.backButtonHandler);
+    }
+
+    componentWillUnmount() {
+        BackButtonUtils.removeBackButtonListener(BackButtonUtils.backButtonHandler);
```

Comment:
I'm not sure what the purpose of passing `backButtonHandler` separately here instead of calling it directly in the libs file is.

Reply:
Yeah maybe just due to the original POC code I left it like that. I can definitely move it to the lib